elasticsearch/src/main/java/com/yahoo/ycsb/db/ElasticSearchClient.java

```diff
@@ -132,6 +144,10 @@ public void init() throws DBException {
       client = node.client();
     }
 
+    //wait for shards to be ready
+    client.admin().cluster()
+      .health(new ClusterHealthRequest("lists").waitForActiveShards(1))
+      .actionGet();
```

Comment:
Both. For local node it requires a short period of time for the shards to be available, otherwise the benchmark operations would start to early and cause UnavailableShardsException. In remote datastore scenario, we should assume the remote shards are available, but if they aren't, benchmarking would fail either way.